packages/apps/graph/generated-schema.graphql

```diff
@@ -47,6 +47,41 @@ type BlockTransactionsConnectionEdge {
   node: Transaction!
 }
 
+"""A transaction cmd."""
+type Cmd {
```

Comment:
Maybe we should make the type name for `Cmd` more specific to what it is in the context of the transaction. I think this would then be `TransactionCommand`